lib/components/IndividualMountain.jsx

```diff
@@ -18,6 +18,7 @@ export default class IndividualMountain extends Component {
 	}
 
 	componentWillMount() {
+    window.scrollTo(0, 0)
```

Comment:
This makes the individual mountain page render at the top instead of the middle of the page that it was doing.